src/scripts.js

```diff
@@ -13,3 +13,73 @@ console.log('This is the JavaScript entry file - your code begins here.');
 
 import userData from './data/users';
 import UserRepository from './UserRepository';
+import User from './User';
+
+//---------------------EVENT LISTENER--------------------------------------//
+window.addEventListener('load', loadPage);
+ 
+//---------------------GLOBAL VARIABLES--------------------------------------//
+let allUserData;
+let allHydrationData;
+let allSleeperData;
+const welcomeName = document.querySelector(".welcome-user");
+const address = document.querySelector(".address");
+const strideLength = document.querySelector(".stride-length-text");
+
+async function loadPage() {
+  const dataSets = await fetchPageData();
+  console.log("dataSets", dataSets);
+  await generateRepoClasses(dataSets);
+  loadPageInfo();
+}
+
+async function fetchPageData() {
+  const userRepoPromise = fetchData('users')
+  .then(console.log('success'))
+  // .catch(err => console.log("error message"));
+
+  const hydrationRepoPromise = fetchData('hydration')
+    .then(console.log('success'));
+      //  .catch(err => console.log(console.log(errorMessage));
+
+  const sleepRepoPromise = fetchData('sleep')
+    .then(console.log('success'));
+      //  .catch(err => /* do something else */);
+
+  const apiDataSets = await Promise.all([userRepoPromise, hydrationRepoPromise, sleepRepoPromise]).then((values) => {
+    return values;
+  });
+
+  return apiDataSets;
+}
+
+function generateRepoClasses(dataSets) {
+  //TO DO: Refactor-loop through argument to dry up.
+  allUserData = new UserRepository(dataSets[0].userData);
+  // allHydrationData = new HydrationRespository(dataSets[1].hydrationData);
+  // allSleeperData = new SleepRepository(dataSets[2].sleepData);
```

Comment:
Love that you kept these comments in for us to add back in later as we add logic! Great job!